pdns/dnsrulactions.hh

```diff
@@ -1301,6 +1301,25 @@ private:
   std::string d_reason;
 };
 
+class TempFailureTTLAction : public DNSAction
```

Comment:
I would like the name of the action to reflect that it only applies to the cache, but something like `TempFailureCacheTTLAction` is quite a mouthful..